package.json

```diff
@@ -41,7 +41,7 @@
     "bench": "node test --bench",
     "lint": "eslint --fix lib/marked.js test/index.js",
     "build": "uglifyjs lib/marked.js -cm  --comments /Copyright/ -o marked.min.js",
-    "preversion": "npm run build"
+    "preversion": "npm run build && (git diff --quiet || git commit -am 'minify')"
```

Comment:
what's this?

Reply:
This was a recommendation from @UziTech. Basically instead of build then commit `min`...let NPM + Git do it for us. As far as the details of what Git is actually doing I would have to read the docs.